templates/guides/import/edgecast_rules_engine_policy.md

```diff
@@ -0,0 +1,26 @@
+# Import
+## edgecast_rules_engine_policy
+
+To import a resource, first write a resource block for it in your configuration, establishing the name by which it will be known to Terraform:
+
+```terraform
+resource "edgecast_rules_engine_policy" "example" {
+  
+}
+```
+
+## Usage
+Now terraform import can be run to attach an existing instance to this resource configuration:
+
+
+```shell
+terraform import edgecast_rules_engine_policy.example ACCOUNT_NUMBER:PORTAL_TYPE_ID:CUSTOMER_USER_ID:ID   
+```
+|                  |                                                                   |
+|:-----------------|-------------------------------------------------------------------|
+| ACCOUNT_NUMBER   | The EdgeCast account number the customer user is associated with. |
+| PORTAL_TYPE_ID   | The portal type id of the cname to import.                        | 
```

Comment:
I just noticed a handful of these descriptions mention cname and edgecname throughout instead of being tailored to the resource type they are referring to.

Reply:
Thanks these have been updated